action.yaml

```diff
@@ -17,8 +17,18 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Download Coder binary from access URL
-      run: curl -fsSL $CODER_URL/bin/coder-linux-amd64 -o /usr/local/bin/coder && chmod +x /usr/local/bin/coder
+    - name: Download and install Coder binary
+      run: |
+        if [ "${{ runner.arch }}" == "ARM64" ]; then
+          ARCH="arm64"
+        else
+          ARCH="amd64"
+        fi
+        mkdir -p "$HOME"/.local/bin
+        curl -fsSL "$CODER_URL"/bin/coder-linux-${ARCH} -o "$HOME"/.local/bin/coder
```

Comment:
Should we add `--compressed` to speed up the download?

Reply:
Yeah good idea. Although GitHub runners are fast but as we are fetching from Coder server it can be on a slow network.